src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -38,12 +32,13 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
             var document = context.Document;
             var span = context.Span;
             var cancellationToken = context.CancellationToken;
-
-            var addImportService = document.GetLanguageService<IAddImportFeatureService>();
+            var diagnostics = context.Diagnostics;
 
             var solution = document.Project.Solution;
             var options = solution.Options;
 
+            var addImportService = document.GetLanguageService<IAddImportFeatureService>();
```

Comment:
why did this move?

Reply:
I'm sure at some point there was a very important reason ;)